docs/.editorconfig

```diff
@@ -11,7 +11,7 @@ trim_trailing_whitespace = true
 indent_style = tab
 indent_size = 8
 
-[*.{tmpl,html}]
+[*.{tmpl, html}]
 indent_style = tab
 indent_size = 4
 
```

Comment:
May want to merge this file with the one in root (and update any files in `docs` to conform).

Reply:
@silverwind done